code/modules/economy/shippingmarket.dm

```diff
@@ -507,7 +507,7 @@
 					if (P && !P.density)
 						P.close()
 
-		shipped_thing.throw_at(target, 100, 1)
+		shipped_thing.throw_at(target, get_dist(spawnpoint, target), 1)
```

Comment:
this should maybe be stored somewhere instead of calculating it for each crate? global? feels weird.

Reply:
static var?

Reply:
implemented in [189ac75](https://github.com/goonstation/goonstation/pull/10780/commits/189ac752ac194d37aa5e3ecc90bd4bd59ef229ab)